### PR TITLE
Unskip gpu tests

### PR DESCRIPTION
**Pull request by Rubtsowa**

### CODE REVIEW

File: tests/skipped_tests_gpu.tbl [hunk 1]
```diff
@@ -37,209 +13,114 @@ tests/test_linalg.py::test_cond[None-[[1, 0, -1], [0, 1, 0], [1, 0, 1]]]
...
+tests/third_party/cupy/fft_tests/test_fft.py::TestFftn_param_20_{axes=None, norm=None, s=None, shape=(2, 3, 4, 5)}::test_fftn
+tests/third_party/cupy/fft_tests/test_fft.py::TestFftn_param_20_{axes=None, norm=None, s=None, shape=(2, 3, 4, 5)}::test_ifftn
+tests/third_party/cupy/fft_tests/test_fft.py::TestRfft_param_0_{n=None, norm=None, shape=(10,)}::test_rfft
+tests/third_party/cupy/fft_tests/test_fft.py::TestRfft_param_1_{n=None, norm=None, shape=(10, 10)}::test_rfft
+tests/third_party/cupy/fft_tests/test_fft.py::TestRfft_param_4_{n=5, norm=None, shape=(10,)}::test_rfft
+tests/third_party/cupy/fft_tests/test_fft.py::TestRfft_param_5_{n=5, norm=None, shape=(10, 10)}::test_rfft
+tests/third_party/cupy/fft_tests/test_fft.py::TestRfft_param_8_{n=10, norm=None, shape=(10,)}::test_rfft
+tests/third_party/cupy/fft_tests/test_fft.py::TestRfft_param_9_{n=10, norm=None, shape=(10, 10)}::test_rfft
+tests/third_party/cupy/fft_tests/test_fft.py::TestRfft_param_12_{n=15, norm=None, shape=(10,)}::test_rfft
+tests/third_party/cupy/fft_tests/test_fft.py::TestRfft_param_13_{n=15, norm=None, shape=(10, 10)}::test_rfft
```
Comment (samir-nasibli) [inline tests/skipped_tests_gpu.tbl:116]: ?

File: tests/skipped_tests_gpu.tbl [hunk 2]
```diff
@@ -319,8 +192,23 @@ tests/third_party/cupy/core_tests/test_ndarray_reduction.py::TestCubReduction_pa
...
+tests/third_party/cupy/creation_tests/test_basic.py::TestBasic::test_empty_like_contiguity3
+tests/third_party/cupy/creation_tests/test_basic.py::TestBasic::test_empty_like_invalid_order
+tests/third_party/cupy/creation_tests/test_basic.py::TestBasic::test_empty_like_subok
+tests/third_party/cupy/creation_tests/test_basic.py::TestBasic::test_empty_zero_sized_array_strides
+tests/third_party/cupy/creation_tests/test_basic.py::TestBasic::test_eye
+tests/third_party/cupy/creation_tests/test_basic.py::TestBasic::test_full_default_dtype
+tests/third_party/cupy/creation_tests/test_basic.py::TestBasic::test_full_like_subok
+tests/third_party/cupy/creation_tests/test_basic.py::TestBasic::test_ones_like_subok
+tests/third_party/cupy/creation_tests/test_basic.py::TestBasic::test_zeros_like_subok
+tests/third_party/cupy/creation_tests/test_basic.py::TestBasic::test_zeros_strides
```
Comment (samir-nasibli) [inline tests/skipped_tests_gpu.tbl:210]: Why are you turning off these tests?

File: tests/skipped_tests_gpu.tbl [hunk 3]
```diff
@@ -447,6 +283,44 @@ tests/third_party/cupy/creation_tests/test_from_data.py::TestFromData::test_asco
...
+tests/third_party/cupy/creation_tests/test_from_data.py::TestArrayCopy_param_14_{copy=False, ndmin=3, xp=numpy}::test_cupy_array
+tests/third_party/cupy/creation_tests/test_from_data.py::TestArrayCopy_param_15_{copy=False, ndmin=3, xp=dpnp}::test_cupy_array
+tests/third_party/cupy/creation_tests/test_ranges.py::TestRanges::test_linspace_array_start_stop
+tests/third_party/cupy/creation_tests/test_ranges.py::TestRanges::test_linspace_array_start_stop_axis1
+tests/third_party/cupy/creation_tests/test_ranges.py::TestRanges::test_linspace_float_underflow
+tests/third_party/cupy/creation_tests/test_ranges.py::TestRanges::test_linspace_mixed_start_stop
+tests/third_party/cupy/creation_tests/test_ranges.py::TestRanges::test_linspace_mixed_start_stop2
+tests/third_party/cupy/creation_tests/test_ranges.py::TestRanges::test_linspace_start_stop_list
+tests/third_party/cupy/creation_tests/test_ranges.py::TestOgrid::test_ogrid4
+tests/third_party/cupy/creation_tests/test_ranges.py::TestOgrid::test_ogrid5
```
Comment (samir-nasibli) [inline tests/skipped_tests_gpu.tbl:323]: ?

**Review by samir-nasibli — CHANGES_REQUESTED** [review]
Please provide description why are you disabling this tests.

The name of the PR does not match their entity.

**Review by shssf — APPROVED** [review]

**Review by samir-nasibli — APPROVED** [review]

**Pull request merged**